### DISCUSSION TITLE
Its always do Ragdoll when player jump and land?

### DISCUSSION THREAD

**Discussion by oivio** [Sixze/ALS-Refactored#361] [Q&A]
I did start to work on sides roller version and did notice always when I make player to jump. It always on landing go in to ragdoll how can I stop it?
I do understand it suppose to be feature for player falling form really high places?
In my case I only make player jump 1m above the ground.

In my opinion it should be optional same as rolling on landing.

**Comment by Sixze** [comment]
This behavior can be controlled in `CS_Als_Default`.
